SPECS/kernel/kernel-uki-dracut.conf

```diff
@@ -31,8 +34,8 @@ drivers+=" virtio-blk virtio-scsi "      # qemu-kvm
 drivers+=" hv-storvsc "                  # hyperv
 drivers+=" xen-blkfront "                # xen
 
-# root encryption
-drivers+=" dm_crypt "
+# coal
+drivers+=" overlay "
```

Comment:
This is not suitable for CVM, which also uses this spec. Let's look into how to conditionally add this driver (and the above dracut module) when building this package. potentially some coal specific macro?